File: LabFinal/LabFinalTask/Command/CommandLog.cs
using LabFinalTask.ENUM;

namespace LabFinalTask.Command;

public class CommandLog
{
    public ICommand Command;
    public COMMANDTYPE Commandtype;
    private string type;
    public DateTime timestamp;

    public CommandLog(ICommand command, COMMANDTYPE commandtype)
    {
        Command = command;
        Commandtype = commandtype;
        timestamp = DateTime.Now;
    }
}

File: TaskManager/task-manager-lab-final/src/main/java/models/CommandLog.java
package models;

import java.security.Timestamp;

public class CommandLog {
    private String description;
    private Timestamp timestamp;
    private CommandType type;

    public CommandLog(String description, Timestamp timestamp, CommandType type){
        this.description = description;
        this.timestamp = timestamp;
        this.type = type;
    }
}
